common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2431,6 +2431,21 @@
     ],
     "sqlState" : "XX000"
   },
+  "INTERNAL_ERROR_INVALID_HIVE_COLUMN_TYPE" : {
+    "message" : [
+      "Failed to convert Hive table to Spark catalog table.",
+      "Error: <message>",
```

Comment:
It's not related to your PR but since we are touching it, can we see if this message is redundant as the causedBy error already contains it?